src/commands/service/create.ts

```diff
@@ -67,6 +75,24 @@ export const fetchValues = (opts: CommandOptions): CommandValues => {
     middlewaresArray = opts.middlewares.split(",").map((str) => str.trim());
   }
 
+  let serviceVgArray: string[] = [];
```

Comment:
There are some cases here we might want to see if we can test at this level
<img width="940" alt="image" src="https://user-images.githubusercontent.com/3119513/79494960-bb48df00-7fd8-11ea-8e0d-6fe8014521af.png">

Reply:
for comma delimited string, it will be nice to add some test like " abc , xyz ", "abc,,def"
thanks

Reply:
Resolved!